packages/nextcloud/test/user_status_test.dart

```diff
@@ -19,6 +19,7 @@ void main() {
         if (Invoker.current!.liveTest.errors.isNotEmpty) {
           print(await container.allLogs());
         }
+        validateFixture(preset);
```

Comment:
Every test now calls validateFixtures.
can't this be done directly in the nextcloud_test package (inside the `presets` function)?

Reply:
Ah true, this can easily be added in a second teardown. With printing the log on failure it was not possible because the container needed to be still around at that time.